### DISCUSSION TITLE
How to use the State component used in the Details and useDetails documentation?

### DISCUSSION THREAD

**Discussion by KrychuTM** [primer/react#1423] [Q&A] [answered]
Is that State component part of Primer React? I can't seem to auto import it nor do I see any documentation about what it is, what its for or how its made. 😭

Link: https://primer.style/react/Details

**Comment by colebemis** [comment, marked as answer]
That component was made for documentation purposes and isn't intended to be used in application code. If you want to use it in your app, you can copy the code from here: https://github.com/primer/react/blob/main/docs/components/State.js

**Reply by KrychuTM** [reply]
Ah thanks alot!
